Servers/UI/OJS.Servers.Ui.Models/Search/SearchRequestModel.cs

```diff
@@ -0,0 +1,21 @@
+namespace OJS.Servers.Ui.Models.Search;
+
+using Microsoft.AspNetCore.Mvc;
+using OJS.Services.Ui.Models.Search;
+using SoftUni.AutoMapper.Infrastructure.Models;
+
+public class SearchRequestModel : IMapTo<SearchServiceModel>
+{
+    public string? SearchTerm { get; set; } = null!;
+
+    [BindProperty(Name = "page")]
+    public int? PageNumber { get; set; }
+
+    public SearchSelectType SelectedTerm { get; set; }
+
+    public int? ItemsPerPage { get; set; }
```

Comment:
Is there a default value if this is not passed?

Reply:
We are handling it with the ValidationService

Reply:
Remove nullables and unused properties.